mne/preprocessing/nirs/tests/test_nirs.py

```diff
@@ -306,3 +310,54 @@ def test_fnirs_channel_naming_and_order_custom_optical_density():
     # and this is how you would fix the ordering, then it should pass
     raw.pick(picks=[0, 3, 1, 4, 2, 5])
     _check_channels_ordered(raw, [760, 850])
+
+
+def test_fnirs_channel_naming_and_order_custom_chroma():
+    """Ensure fNIRS channel checking on manually created data."""
+    data = np.random.normal(size=(6, 10))
```

Comment:
I know it doesn't currently matter since you don't use the data for anything, but let's use a seeded random var in case we later change the test to use the data (since we always try to do it, it's reasonable to assume it has been done here):
```suggestion
    data = np.random.RandomState(0).randn(6, 10)
```
Or if you want it to be simpler you can just use `np.zeros(...)`